### DISCUSSION TITLE
I've created ArchLinux packages

### DISCUSSION THREAD

**Discussion by mmozeiko** [sct/overseerr#249] [General]
I've created ArchLinux packages in AUR for overseerr.

Built from tag: https://aur.archlinux.org/packages/overseerr/
Built from latest git: https://aur.archlinux.org/packages/overseerr-git/

Ping me if you need or want something updated there.

To install these just use your favorite AUR package manager:
```
yay -S overseer
```

**Comment by sct** [comment]
This is great! I have no way to test it but perhaps if others are having success with this we can eventually add it to the README.

**Comment by samwiseg0** [comment]
@mmozeiko This was added to the wiki! Thanks! https://github.com/sct/overseerr/wiki/Installation
